packages/libs/wdk-client/src/Components/Overlays/Tooltip.tsx

```diff
@@ -23,151 +16,58 @@ let defaultOptions = {
-    if (this.api == null) {
-      this._setupTooltip(nextProps);
-      return;
-    }
-
-    if (nextProps.open != null) {
-      this.api.toggle(nextProps.open);
-    }
-
-    if (nextProps.classes != null) {
-      this.api.set('style.classes', nextProps.classes);
-    }
-
-    if (nextProps.showEvent != null) {
-      this.api.set(
-        'show.event',
-        nextProps.open == null ? nextProps.showEvent : false
-      );
-    }
-
-    if (nextProps.hideEvent != null) {
-      this.api.set(
-        'hide.event',
-        nextProps.open == null ? nextProps.hideEvent : false
-      );
-    }
-
-    if (nextProps.position != null) {
-      this.api.set('position.at', nextProps.position.at);
-      this.api.set('position.my', nextProps.position.my);
-    }
-
-    this.api.reposition();
-  }
-
-  componentWillUnmount() {
-    this._destroyTooltip();
-  }
-
-  _setupTooltip(props: Props) {
-    let {
-      content,
-      open,
-      showEvent,
-      showDelay = defaultOptions.show.delay,
-      hideEvent,
-      hideDelay = defaultOptions.hide.delay,
-      classes = 'qtip-wdk',
-      position = defaultOptions.position,
-      solo = true,
-      showTip = true,
-    } = props;
-
-    if (content == null) {
-      return;
-    }
-
-    if (open != null && (showEvent != null || hideEvent != null)) {
-      console.warn(
-        'Tooltip was passed props `open` and either `showEvent` or `hideEvent`. ' +
-          'Since `open` was provided, `showEvent` and `hideEvent` will be ignored.'
-      );
-    }
-
-    this.api = $(ReactDOM.findDOMNode(this) as HTMLElement)
-      .qtip({
-        content: { text: $(this.contentContainer) },
-        style: { classes, tip: { corner: showTip } },
-        show: {
-          ...defaultOptions.show,
-          solo,
-          event: open == null ? showEvent : false,
-          delay: showDelay,
-        },
-        hide: {
-          ...defaultOptions.hide,
-          event: open == null ? hideEvent : false,
-          delay: hideDelay,
-        },
-        position,
-        events: {
-          show: (event) => {
-            if (props.onShow) props.onShow(event);
-          },
-          hide: (event) => {
-            if (props.onHide) props.onHide(event);
-          },
-        },
-      })
-      .qtip('api');
-
-    if (open != null) this.api.toggle(open);
-  }
-
-  _destroyTooltip() {
-    $(ReactDOM.findDOMNode(this) as HTMLElement).qtip('destroy');
-  }
-
-  render() {
-    return (
-      <React.Fragment>
-        {React.Children.only(this.props.children)}
-        {ReactDOM.createPortal(this.props.content, this.contentContainer)}
-      </React.Fragment>
-    );
-  }
-}
-
-export const EagerlyLoadedTooltip = wrappable(Tooltip);
-
-const withJquery = lazy<Props>(async () => {
-  // @ts-ignore
-  await import('!!script-loader!../../../vendored/jquery.qtip.min.js');
-});
+const WDKClientTooltip = ({
+  content,
+  children,
+  classes = 'qtip-wdk',
```

Comment:
Looks like we need to update the CSS applied to this class in `wdk-Common.scss`
![image](https://github.com/VEuPathDB/web-monorepo/assets/69446567/d1f7db99-ea02-4db1-8c3c-40c35edcc81c)

Reply:
After some digging, I found that if I pass in `classes = ''` to `WDKClientTooltip` in this instance- thereby bypassing the default `qtip-wdk` class assignment- then the icon renders as expected _**but**_ the tooltip doesn't render. I'm not able to figure out a quick solution to this problem.

Reply:
I would even go so far to say that maybe we don't need this component at all. Thoughts?

Reply:
@dmfalke I agree